src/keycloak/urls_patterns.py

```diff
@@ -121,6 +121,9 @@
 URL_ADMIN_CLIENT_AUTHZ_RESOURCE_BASED_PERMISSION = (
     URL_ADMIN_CLIENT_AUTHZ + "/permission/resource?max=-1"
 )
+URL_ADMIN_CLIENT_AUTHZ_SCOPE_BASED_PERMISSION = (
+    URL_ADMIN_CLIENT + "/authz/resource-server/permission/scope?max=-1"
```

Comment:
Can we utilize `URL_ADMIN_CLIENT_AUTHZ` instead of `URL_ADMIN_CLIENT`?

Reply:
Yes, I updated the file to use it